packages/eslint-config-nori/package.json

```diff
@@ -30,18 +30,18 @@
     "react-dom": "~17"
   },
   "dependencies": {
-    "@types/eslint": "8.4.3",
     "@fintechstudios/eslint-plugin-chai-as-promised": "3.1.0",
-    "@graphql-eslint/eslint-plugin": "^3.19.1",
```

Comment:
Didn't **need** to bump this, but I was seeing an eslint error in VSCode which caused me to see if bumping this would fix the error. This bump did not fix it unfortuately, however the error ended up being unrelated to my issue and it turned out this error exists on master as well (the error seems to be gracefully handled and related to parsing the new ts `satisfies` operator).